gitpod-io/gitpod
# Revise Terms

#### How to test
1. TC – login **and** signup should work as usual
2. TC – run sql `update d_b_terms_acceptance_entry set termsRevision='2020-10';` in order to test an update. 
  (feel free to ask me to run it for you 🙏🏻 )
  then:
 2.1. `/workspaces` page must not be affected, neither any running workspace
 2.2. starting new/existing workspaces must be blocked by `We've update ToS` page
 2.3. re-login should be blocked by `We've update ToS` page
